app.js

```diff
@@ -0,0 +1,15 @@
+// EasyScore API Example
+
```

Comment:
What's this file used for?

Reply:
I was using this to test things out in the original VexFlow library & compare it to the output of the web components. I'll remove it